### PR TITLE
Add trunc mode div

### PR DESCRIPTION
**Pull request by bszmelcz**
## Details
`xfail` tests for which inputs dtypes differ

### CODE REVIEW

File: tests/layer_tests/pytorch_tests/test_div.py [hunk 1]
```diff
@@ -54,7 +52,38 @@ def forward(self, input_tensor, other_tensor):
...
+    def test_div_float(self, input_array, input_type, other_array, other_type, rounding_mode, ie_device, precision, ir_version):
+        self.input_array = input_array
+        self.input_type = input_type
+        self.other_array = other_array
+        self.other_type = other_type
+        self._test(*self.create_model(rounding_mode), ie_device, precision, ir_version)
+
+
+    @pytest.mark.parametrize(("input_array", "other_array"), [
+        [np.random.rand(5, 5), np.random.rand(1)],
```
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/test_div.py]: You also need to restrict division by zero

Comment (bszmelcz) [reply tests/layer_tests/pytorch_tests/test_div.py]: @mvafin Done, changed to `np.random.uniform` with `low=0.1`.

File: tests/layer_tests/pytorch_tests/test_div.py [hunk 2]
```diff
@@ -27,11 +27,11 @@ def forward(self, input_tensor, other_tensor):
         return aten_div(rounding_mode), ref_net, "aten::div"
 
     @pytest.mark.parametrize(("input_array", "other_array"), [
-        [np.random.rand(5, 5), np.random.rand(1)],
-        [np.random.rand(5, 5, 1), np.random.rand(1)],
-        [np.random.rand(1, 1, 5, 5), np.random.rand(1)],
-        [np.random.rand(5, 5, 1), np.random.rand(5, 1)],
-        [np.random.rand(5, 5), np.random.rand(5, 5)],
+        [np.random.uniform(low=0.1, high=1.0, size=(5, 5)), np.random.rand(1)],
```
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/test_div.py]: I think you changed the wrong array, you should have changed divisor

Comment (bszmelcz) [reply tests/layer_tests/pytorch_tests/test_div.py]: Oh yeah you are right. Fixed.

**Comment by mvafin** [comment]
Fixed in PR #93

**Pull request closed without merging**